src/modules/land_detector/LandDetector.h

```diff
@@ -162,10 +149,17 @@ class LandDetector : public ModuleBase<LandDetector>, ModuleParams, px4::Schedul
 
 	actuator_armed_s         _actuator_armed{};
 	vehicle_acceleration_s   _vehicle_acceleration{};
-	vehicle_land_detected_s  _land_detected{};
+	vehicle_land_detected_s _land_detected = {
+		.timestamp = 0,
+		.alt_max = -1.0f,
+		.freefall = false,
+		.ground_contact = true,
+		.maybe_landed = true,
+		.landed = true,
+	};
```

Comment:
missing setting of in_ground_effect.
Not used for multicopter, but seems better to set it too.

Reply:
It's used for multicopter but in the estimator to stop fusing affected baro data into the altitude.